qt/app.py

```diff
@@ -1182,11 +1182,26 @@ def updateTimeLine(self, refreshSnapshotsList = True):
                 item = self.timeLine.addSnapshot(sid)
             self.timeLine.checkSelection()
 
+    def validateOnTakeSnapshot(self):
+        hasMissing, missing = snapshots.hasMissing(self.config.include())
+        if hasMissing:
+            msgMissing = '\n'.join(missing)
+            msg = f'{_("The following folders are missing")}:\n\n{msgMissing}\n\n{_("Do you want to proceed?")}'
+            answer = messagebox.warningYesNo(self, msg)
+            return answer == QMessageBox.StandardButton.Yes
+        return True
+
     def btnTakeSnapshotClicked(self):
+        proceed = self.validateOnTakeSnapshot()
+        if not proceed:
+            return
         backintime.takeSnapshotAsync(self.config)
         self.updateTakeSnapshot(True)
 
     def btnTakeSnapshotChecksumClicked(self):
+        proceed = self.validateOnTakeSnapshot()
+        if not proceed:
+            return
         backintime.takeSnapshotAsync(self.config, checksum = True)
         self.updateTakeSnapshot(True)
```

Comment:
Don't hesitate doing a little refactoring. 😄 

```suggestion
    def btnTakeSnapshotClicked(self):
        _take_snapshot_clicked(checksum=False)

    def btnTakeSnapshotChecksumClicked(self):
        _take_snapshot_clicked(checksum=True)

    def _take_snapshot_clicked(self, checksum):
        if not self.validateOnTakeSnapshot():
            return

        backintime.takeSnapshotAsync(self.config, checksum=checksum)
        self.updateTakeSnapshot(True)
```